students/briannagata/session03/mailroom/actions.py

```diff
@@ -0,0 +1,30 @@
+# Modules to process selections made in mailroom.py
+# Python 3.6
+
+
+def list_database(donors):
+    """ Print the list of donor names"""    
+    for name in donors.keys():
```

Comment:
```
for name in donors:
...
```

Reply:
Yep, for accessing the keys you actually don't need to call keys(). The keys() function is useful for providing a set-like object, which you could use for comparing the keys of different dictionaries, for example.